src/clj/rems/api/schema.clj

```diff
@@ -47,11 +47,21 @@
    :end (s/maybe DateTime)
    :mail s/Str})
 
+;; TODO: Remove once all admin APIs have separate endpoints for enabled
+;;       and archived.
 (s/defschema UpdateStateCommand
   {:id s/Int
    :enabled s/Bool
    :archived s/Bool})
 
+(s/defschema EnabledCommand
+  {:id s/Int
+   :enabled s/Bool})
```

Comment:
A comment about the design decision to have separate commands for enabling and archiving:

The UI components ('Enable'/'Disable' etc. buttons) currently rely on the item (workflow etc.) having :enabled and :archived keys. If the backend had one command with, e.g., :id and :value fields, the UI would need to map :enabled -> :value, and :archived -> :value, which backend or database query would then map back to :enabled and :archived before updating the db. Having two separate commands avoids that round trip, so the pipe between UI and the database stays cleaner.